iroh-net/src/disco.rs

```diff
@@ -143,6 +143,8 @@ pub struct Pong {
 pub struct CallMeMaybe {
     /// What the peer believes its endpoints are.
     pub my_number: Vec<SocketAddr>,
+    /// If true invalidate all previous pongs. This is sent after a total state reset.
+    pub clear_pongs: bool,
```

Comment:
Can't we infer if the pongs need to be cleared somehow?

If we really have to have this field, I would prefer to have the field name be a description of why this `CallMeMaybe` is different than the normal ones rather than an instruction about what to do...

Reply:
> Can't we infer if the pongs need to be cleared somehow?
I spend some time thinking about this and didn't find anything. Only idea I had was lowering the maximum age of the most recent pong of the best_addr for it to be considered valid (currently 6 seconds). Other ideas welcome.

Reply:
Moved `clear_pongs` feature to #1742